js/my.js

```diff
@@ -232,3 +232,25 @@ document.addEventListener("DOMContentLoaded", function () {
   });
 });
 
+function parseCommand(client, nick, host, cmd) {
+  var args = cmd.split(' ');
+  args[0] = args[0].replace(/^\//, '');
+
+  if (args && args[0] === 'query' && args[1]) {
+    var to = args[1];
+    var msg = privMSG[to];
+
+    if (!msg) {
+      msg = privMSG[to] = new Tab({
+        chan: to,
+        client: client,
+        nick: nick,
+        host: host,
+      });
+    }
+
+    return;
+  }
+
+  client.send(args.shift(), args.shift(), args.join(' '));
+}
```

Comment:
I think we should:
* not send anything if the command is prefixed with `/` but not supported by us yet.
* don't replace then compare to `query` when we can just compare to `/query`.
* if we don't have a tab we create one, but if we do, we should make that tab the active one.